From 1440239102fe5ef1a1781d7d073d21c06469e6a0 Mon Sep 17 00:00:00 2001
From: pmandri <padmashree.mandri@intel.com>
Date: Wed, 25 Aug 2021 11:33:18 +0530
Subject: [PATCH] Set Audio Volume to Maximun

Set Audio volume for STREAM_MUSIC to maximun

Tracked-On:
Signed-off-by: pmandri <padmashree.mandri@intel.com>
---
 services/core/java/com/android/server/audio/AudioService.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/core/java/com/android/server/audio/AudioService.java b/services/core/java/com/android/server/audio/AudioService.java
index fb9cf0dc5f06..abfd6a0090fb 100755
--- a/services/core/java/com/android/server/audio/AudioService.java
+++ b/services/core/java/com/android/server/audio/AudioService.java
@@ -738,7 +738,7 @@ public class AudioService extends IAudioService.Stub
                     (MAX_STREAM_VOLUME[AudioSystem.STREAM_VOICE_CALL] * 3) / 4;
         }
 
-        int maxMusicVolume = SystemProperties.getInt("ro.config.media_vol_steps", -1);
+        int maxMusicVolume = 15;
         if (maxMusicVolume != -1) {
             MAX_STREAM_VOLUME[AudioSystem.STREAM_MUSIC] = maxMusicVolume;
         }
-- 
2.17.1

